### PR TITLE
Games screen tableview

### PR DESCRIPTION
**Pull request by sudo-ulmas**
[//]: # (Feel free to remove parts that do not relate to the changes you are willing to bring into main)

### Games screen table view
[//]: # (Give a brief summary of why the change is being implemented, i.e. feature was built, bug fixing and etc.)

### Changes
[//]: # (List change details here for Code Reviewers)
1. change navbar color across all screen
2. create custom table view cell
3. implement table view
4. add mock data

[//]: # (Use the table below if you have screenshots/videos to show visual updates)

Simulator Screen Shot - iPhone 13 mini - 2023-01-18 at 17 25 30

### CODE REVIEW

File: SteamList/Extensions/ViewControllerExtensions.swift [hunk 1]
```diff
@@ -9,8 +9,8 @@ import UIKit
 
 extension UIViewController {
     func applyBackgroundGradient() {
-        let firstColor = UIColor(red: 41/255, green: 92/255, blue: 140/255, alpha: 1).cgColor
-        let secondColor = UIColor(red: 40/255, green: 72/255, blue: 95/255, alpha: 1).cgColor
+        let firstColor = UIColor(red: 0.132, green: 0.241, blue: 0.337, alpha: 1).cgColor
```
Comment (AbboskhonShukurullaev) [inline SteamList/Extensions/ViewControllerExtensions.swift:12]: It's your personal choice how to make it, but I would prefer the division style as it's more convenient for me

**Comment by AbboskhonShukurullaev** [comment]
should show no tint color when tapped

Screenshot 2023-01-18 at 17 40 06

**Comment by AbboskhonShukurullaev** [comment]
Could you make your UI as neat as it was in when u used storyboards?

**Comment by AbboskhonShukurullaev** [comment]
Could u implement star button taps as well?

File: SteamList/Extensions/ViewControllerExtensions.swift (hunk 1 above)
Comment (sudo-ulmas) [reply SteamList/Extensions/ViewControllerExtensions.swift:12]: I also like the division style, I just tested floating point style

**Review by AbboskhonShukurullaev — APPROVED** [review]

**Pull request merged**